### ISSUE TITLE
Paste Style on Container changes link destination in additional options

### ISSUE THREAD

**Issue by WestEndian** [elementor/elementor#21954]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I have four Containers with identical styles. Each container uses the Additional Options -> Link option to link to a distinct page on my site. 

1. I updated the style of one container.
2. I used the Copy function in the Navigator panel to copy that  container. 
3. I used the Paste Style function in the Navigator panel to paste the style onto one of the other containers. 
4. The style was copied successfully. But the link in the second container was also changed. 

I would expect the link to remain unchanged as it is not a style element. 

See screencast: https://user-images.githubusercontent.com/129726940/230313617-de8b1e91-e279-4b73-a050-f35a896f7156.mp4

### Steps to reproduce

1. Add multiple containers to a page and add some content
2. Add different links to each container in Additional Options -> Link
3. Update the style of one container.
4. Use the Copy function in the Navigator panel to copy that container. 
5. Use the Paste Style function in the Navigator panel to paste the style onto one of the other containers. 
6. Check the link 

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v10.3.34
	PHP Version: 7.4.28
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 80M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://staging.kids.org.uk
	Home URL: https://staging.kids.org.uk
	WP Multisite: No
	Max Upload Size: 80 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /hub-content/%postname%/
	Language: en-GB
	Timezone: 0
	Admin Email: nick.balneaves@kids.org.uk
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.3 Safari/605.1.15

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.1.3
		Author: WP Engine

	Code Snippets
		Version: 3.3.0
		Author: Code Snippets Pro

	Custom Post Type UI
		Version: 1.13.5
		Author: WebDevStudios

	Disable Gutenberg
		Version: 2.9
		Author: Jeff Starr

	Elementor
		Version: 3.12.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.1
		Author: Elementor.com

	Koko Analytics
		Version: 1.0.37
		Author: ibericode

	One Click Accessibility
		Version: 2.1.0
		Author: One Click Accessibility

	Password Protected
		Version: 2.6.2
		Author: WPExperts

	UpdraftPlus - Backup/Restore
		Version: 2.23.3.26
		Author: UpdraftPlus.Com, DavidAnderson

	User Menus
		Version: 1.3.1
		Author: Code Atlantic

	Wordfence Assistant
		Version: 1.0.9
		Author: Wordfence

	Wordfence Security
		Version: 7.9.2
		Author: Wordfence

	WP-Optimize - Clean, Compress, Cache
		Version: 3.2.14
		Author: David Anderson, Ruhani Rabin, Team Updraft

	WPForms
		Version: 1.8.1.1
		Author: WPForms

	WPForms Surveys and Polls
		Version: 1.11.0
		Author: WPForms

	WP Mail SMTP Pro
		Version: 3.7.0
		Author: WPForms

== Elements Usage ==
	
	container : 3
		button : 2
		container : 34
		heading : 18
		icon-list : 4
		image : 16
		table-of-contents : 2
		text-editor : 21
		toggle : 2
		video : 2
		wpforms : 2
	footer : 2
		container : 11
		icon-list : 2
		image : 8
		text-editor : 8
	header : 2
		container : 26
		html : 2
		image : 22
		text-editor : 18
	page : 2
		button : 3
		container : 12
		form : 1
		heading : 5
		html : 2
		image : 5
		image-box : 1
		table-of-contents : 1
		text-editor : 7
		video : 1
	popup : 7
		button : 7
		container : 18
		heading : 7
		icon : 1
		image : 6
		text-editor : 7
	wp-post : 3
		button : 2
		container : 53
		heading : 22
		icon-list : 5
		image : 21
		menu-anchor : 1
		table-of-contents : 3
		text-editor : 31
		toggle : 1
		video : 4
		wpforms : 3
	wp-page : 20
		animated-headline : 1
		button : 13
		container : 127
		form : 4
		heading : 35
		html : 7
		icon : 20
		icon-list : 2
		image : 43
		image-box : 1
		login : 1
		loop-grid : 2
		table-of-contents : 2
		text-editor : 51
		theme-page-title : 2
		theme-post-title : 2
		video : 1
		wpforms : 1
	single-post : 3
		container : 3
		image : 2
		menu-anchor : 2
		theme-post-content : 2
		theme-post-excerpt : 1
		theme-post-title : 1
	single-page : 1
		container : 1
		menu-anchor : 1
		theme-post-content : 1
	loop-item : 3
		button : 2
		container : 3
		theme-post-excerpt : 2
		theme-post-featured-image : 2
		theme-post-title : 2

== Settings ==
	
	cpt_support: post, page, participation_story
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1

== Features ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrations ==
	

== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved asset loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Active
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default

== Log ==
	
Log: showing 20 of 292023-03-29 09:56:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.6',
  'to' => '3.11.7',
)]
2023-03-29 17:08:49 [info] elementor::elementor_updater Started 
2023-03-29 17:08:49 [info] Elementor/Upgrades - _on_each_version Start  
2023-03-29 17:08:49 [info] Elementor/Upgrades - _on_each_version Finished 
2023-03-29 17:08:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.0',
)]
2023-03-29 17:08:49 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.0',
)]
2023-03-30 10:29:31 [info] elementor-pro::elementor_pro_updater Started 
2023-03-30 10:29:31 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-30 10:29:31 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-30 10:29:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.0',
)]
2023-03-30 10:29:31 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.0',
)]
2023-04-02 16:50:03 [info] elementor::elementor_updater Started 
2023-04-02 16:50:03 [info] Elementor/Upgrades - _on_each_version Start  
2023-04-02 16:50:04 [info] Elementor/Upgrades - _on_each_version Finished 
2023-04-02 16:50:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.0',
  'to' => '3.12.1',
)]
2023-04-02 16:50:04 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.0',
  'to' => '3.12.1',
)]
2023-04-03 06:48:28 [info] elementor-pro::elementor_pro_updater Started 
2023-04-03 06:48:28 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-03 06:48:28 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-03 06:48:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.0',
  'to' => '3.12.1',
)]

JS: showing 6 of 6JS: 2023-03-29 09:59:46 [error X 8][https://staging.kids.org.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.5:3:656177] elementorFrontend.elements.window.jQuery is not a function 
JS: 2023-03-30 18:21:48 [error X 1][https://staging.kids.org.uk/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.12.0:2:63837] undefined is not an object (evaluating 'this.getEditorControlView("template_id").createTemplate') 
JS: 2023-03-30 19:16:08 [error X 2][https://staging.kids.org.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.0:3:1081266] null is not an object (evaluating '$.getBoundingClientRect') 
JS: 2023-03-31 11:15:48 [error X 3][https://staging.kids.org.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.0:3:673305] this.model.isValidChild is not a function. (In 'this.model.isValidChild(C)', 'this.model.isValidChild' is undefined) 
JS: 2023-03-31 17:16:23 [error X 8][https://staging.kids.org.uk/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.12.0:2:5008] undefined is not an object (evaluating 'e.attributes') 
JS: 2023-04-04 23:56:26 [error X 7][https://staging.kids.org.uk/near-me/:429:49] jQuery(...).value is not a function 

PHP: showing 7 of 7PHP: 2023-03-29 10:54:30 [notice X 27][/bitnami/wordpress/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-30 16:56:22 [notice X 2][/bitnami/wordpress/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-30 19:32:17 [notice X 16][/bitnami/wordpress/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined index: condition_type [array (
  'trace' => '
#0: /bitnami/wordpress/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /bitnami/wordpress/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /bitnami/wordpress/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /bitnami/wordpress/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-03-30 19:32:45 [notice X 1][/bitnami/wordpress/wp-content/plugins/elementor/includes/embed.php::260] Undefined property: stdClass::$title [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-31 11:20:10 [notice X 8][/bitnami/wordpress/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::168] Undefined index: editor_post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-31 17:04:52 [error X 1][/bitnami/wordpress/wp-content/plugins/elementor-pro/modules/popup/module.php::122] Uncaught Error: Call to undefined method Elementor\Core\Kits\Documents\Kit::save_display_settings_data() in /bitnami/wordpress/wp-content/plugins/elementor-pro/modules/popup/module.php:122
Stack trace:
#0 [internal function]: ElementorPro\Modules\Popup\Module->save_display_settings()
#1 /bitnami/wordpress/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#2 /opt/bitnami/wordpress/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#3 /opt/bitnami/wordpress/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#4 /opt/bitnami/wordpress/wp-includes/plugin.php(517): WP_Hook->do_action()
#5 /opt/bitnami/wordpress/wp-admin/admin-ajax.php(188): do_action()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-02 22:18:45 [warning X 1][/bitnami/wordpress/wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/icons/custom-icons.php::199] unlink(Map_web_icon.svg): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==

**Comment by simonclay** [comment]
Just been caught out by this problem. It still exists.
